sys/include/xtimer/implementation.h

```diff
@@ -172,6 +172,11 @@ static inline void xtimer_spin(xtimer_ticks32_t ticks) {
     _xtimer_spin(ticks.ticks32);
 }
 
+static inline void xtimer_msleep(uint32_t milliseconds)
+{
+    _xtimer_tsleep32(_xtimer_ticks_from_usec(milliseconds * US_PER_MS));
```

Comment:
```suggestion
    _xtimer_tsleep64(_xtimer_ticks_from_usec64(milliseconds * US_PER_MS));
```

I the above makes more sense. Otherwise `milliseconds * US_PER_MS` can silently overflow 32 bit and the sleep duration would be much lower. Especially since the `ztimer_xtimer_compat` version would work up until (including) `UINT32_MAX`, it would be surprising to have `xtimer` not supporting this.